com.CalculaPro.app/com.CalculaPro.app.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "com.CalculaPro.app",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "calcula-pro",
+    "modules": [
+        {
+            "name": "Calcula Pro",
+            "buildsystem": "simple",
+            "build-commands": [
+                "dotnet publish -r linux-x64 -c "Release" --output "/home/alejandro/Projects/Calcula-Pro/OPS/obj/desktop/linux/bin" /p:PublishReadyToRun=true /p:PublishSingleFile=true --self-contained"
```

Comment:
so what happen on `aarch64`?

Reply:
also even without knowing how this works I can tell that the command lime is incorrect as it specify a path that doesn't exist, and the `"` aren't escaped making the JSON invalid.